jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -715,3 +715,53 @@ EOF
     echo "Expected: ${expected}"
     [[ "${JBOSS_KIE_ARGS}" == "${expected}" ]]
 }
+
+@test "Verify the Kafka JBPM Emitter" {
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED="true"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_BOOTSTRAP_SERVERS="localhost:9093"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID="jbpmapp"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS="3"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS="2100"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT="dd-MM-yyyy'T'HH:mm:ss.SSSZ"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_PROCESSES_TOPIC_NAME="my-processes-topic"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_TASK_TOPIC_NAME="my-task-topic"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CASES_TOPIC_NAME="my-cases-topic"
+
+    configure_kafka_jbpm_emitter
+
+    local expected=" -Dorg.kie.jbpm.event.emitters.kafka.bootstrap.servers=localhost:9093 -Dorg.kie.jbpm.event.emitters.kafka.acks=3 -Dorg.kie.jbpm.event.emitters.kafka.client.id=jbpmapp -Dorg.kie.jbpm.event.emitters.kafka.max.block.ms=2100 -Dorg.kie.jbpm.event.emitters.kafka.date_format=dd-MM-yyyy'T'HH:mm:ss.SSSZ -Dorg.kie.jbpm.event.emitters.kafka.topic.processes=my-events-topic -Dorg.kie.jbpm.event.emitters.kafka.topic.tasks=my-task-topic -Dorg.kie.jbpm.event.emitters.kafka.topic.cases=my-case-topic"
+    echo "  Result: ${JBOSS_KIE_ARGS}"
+
+    echo "Expected: ${expected}"
+    [[ "${JBOSS_KIE_ARGS}" == "${expected}" ]]
+}
+
+@test "Verify the Kafka JBPM Emitter without bootstrap emitter" {
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED="true"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID="jbpmapp"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS="3"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS="2100"
+    export KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT="dd-MM-yyyy'T'HH:mm:ss.SSSZ"
+
+    configure_kafka_jbpm_emitter
+
+    local expected=""
+    echo "  Result: ${JBOSS_KIE_ARGS}"
+
+    echo "Expected: ${expected}"
+    [[ "${JBOSS_KIE_ARGS}" == "${expected}" ]]
+}
```

Comment:
Is this test passing?? I think it shouldn't as in this tests is set MAX_BLOCK_MS and it is not check as expected. According configure.sh script it should be set all the times https://github.com/jboss-container-images/jboss-kie-modules/pull/460/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cR815

Reply:
in this case not, it just check if the emitter conf is not set at all if there is no bootstrap servers.
Maybe it worth to check the log message instead as expected result.

Reply:
It simulate the fail  if the bootstrapserver pro isn't set

Reply:
@desmax74 can you verify here if the log message is correctly printed?

Reply:
yep